src/main/java/frc/robot/Robot.java

```diff
@@ -135,7 +131,15 @@ public Command getAutonomousCommand() {
     return shuffleboard.getAutonomousCommand();
   }
 
-  public static boolean isTestMode() {
-    return isTestMode;
+  /**
+   * Whether we are currently in Test Mode.
+   * @deprecated use RobotBase.isTest()
+   * Also confused about the Robot being STATIC.
+   * Main builds a single INSTANCE of the robot.
+   * At a later time, it will call the robotInit() method.
```

Comment:
That's true. Honestly at this stage in development I don't think we need to deprecate the method we can just change it. Probably at any stage of development as we can coordinate with the rest of the team. I also think including comments like that are not a good idea on the main branch, the main branch should be more or less finalized. And when hovering over a function the javadoc should describe the function, not thoughts about the implementation of robot.java. 
As for the static/instance issue, I can add a getInstance() method which can be used by Main.java and for getting the test mode. The robot class is a singleton.